googlemock/include/gmock/gmock-matchers.h

```diff
@@ -3614,6 +3614,10 @@ BoundSecondMatcher<Tuple2Matcher, Second> MatcherBindSecond(
   return BoundSecondMatcher<Tuple2Matcher, Second>(tm, second);
 }
 
+// Joins a vector of strings as if they are fields of a tuple; returns
+// the joined string.
+GTEST_API_ string JoinAsTuple(const Strings& fields);
```

Comment:
@davidben I am missing where this is used?

Reply:
It's used in gmock-matchers_test.cc by replicating the declaration. My bad, I forgot to remove the duplicate one.

Reply:
@davidben sorry, I still see it here, I thought we were removing it?